src/route-href.js

```diff
@@ -42,7 +42,13 @@ export class RouteHref {
         }
 
         let href = this.router.generate(this.route, this.params);
-        this.element.setAttribute(this.attribute, href);
+
+        if (this.element.au.controller) {
+          this.element.au.controller.viewModel.href = href;
```

Comment:
I think we will want to use the attribute to index the view model here instead of hardcoding href.